### DISCUSSION TITLE
Support beginRefundRequest

### DISCUSSION THREAD

**Discussion by codal-mpawar** [hyochan/react-native-iap#2480] [Ideas]
Need refund support.

**Comment by codal-mpawar** [comment]
https://developer.apple.com/documentation/storekit/transaction/3803220-beginrefundrequest/

https://developer.apple.com/documentation/storekit/in-app_purchase/original_api_for_in-app_purchase/handling_refund_notifications

**Comment by cervebar** [comment]
It is supported (at least in version `12.10.7`). 
```
import {IapIosSk2} from 'react-native-iap'
....
IapIosSk2.beginRefundRequest(sku)
```
